src/Engine/Utils.java

```diff
@@ -446,6 +446,12 @@ public int compare(SearchNode o1, SearchNode o2)
     return industries;
   }
 
+  /** Calculate the Manhattan distance between the two coordinates (number of moves for a flying unit). */
+  public static int findManhattanDistance(XYCoord c1, XYCoord c2)
+  {
+    return Math.abs(c1.xCoord - c2.xCoord) + Math.abs(c1.yCoord - c2.yCoord);
+  }
```

Comment:
You uh... you do realize we have a function on XYCoord for this, right?

Reply:
I thought I had made one of those at one point, but I couldn't find it when I went looking.